src/EventSubscriber/DefaultCDF.php

```diff
@@ -10,7 +10,7 @@
 /**
  * The Configuration entity CDF creator.
  *
- * @see \Drupal\acquia_contenthub\Event\CreateCDFEntityEvent
+ * @see \Drupal\acquia_contenthub\Event\CreateCdfEntityEvent
  */
 class DefaultCDF implements EventSubscriberInterface
```

Comment:
And DefaultCDF?

Reply:
Out of scope. Please see my overall comment on this PR.